files/conf/k8s-addons

```diff
@@ -134,12 +134,22 @@ do
 done
 
 {{ if .EnableAWSCNI -}}
-echo "Waiting for aws-node to be ready..."
-kubectl -n kube-system -l app.kubernetes.io/name=aws-node wait --for=condition=Ready --timeout=10m pods
+while
+    echo "Waiting for aws-node to be ready..."
+    kubectl -n kube-system -l k8s-app=aws-node wait --for=condition=Ready --timeout=1m pods
+    [ "$?" -ne "0" ]
+do
+    echo "aws-node was not ready after 1 minute"
+done
```

Comment:
How about adding a message that it is ready now?

Reply:
Good call. Done.